src/_guides/language/language-tour.md

```diff
@@ -3862,9 +3863,30 @@ print(names is List<String>); // true
 ### Restricting the parameterized type
 
 When implementing a generic type,
-you might want to limit the types of its parameters.
+you might want to limit the types of its parameters
```

Comment:
This kind of mixes categories.  I don't want to be too technical, but it's probably worth trying for as much accuracy as possible here.  I'd say something like "limit the types that can be provided as arguments to be a subtype of a particular type".  The point being that the constraint is on arguments (not parameters).